common/app/views/fragments/submeta.scala.html

```diff
@@ -25,48 +24,52 @@
             }
 
-            @if(!(content.content.isImmersive && content.content.tags.isArticle)) {
-                <a class="submeta2__link submeta2__link--first"
-                    data-link-name="article section"
-                    href="@LinkTo {/@content.content.sectionLabelLink}">
-                        @Html(Localisation(content.content.sectionLabelName))
-                </a>
-            }
+            <div class="submeta2__links">
+                @if(!(content.content.isImmersive && content.content.tags.isArticle)) {
+                    <a class="submeta2__link submeta2__link--first"
+                        data-link-name="article section"
+                        href="@LinkTo {/@content.content.sectionLabelLink}">
+                            @Html(Localisation(content.content.sectionLabelName))
+                    </a>
+                }
 
-            @content.content.blogOrSeriesTag.map { series =>
-                <a class="submeta2__link" href="@LinkTo {/@series.id}">@series.name</a>
-            }.getOrElse {
-                @if(content.content.isFromTheObserver) {
-                    <a class="submeta2__link" href="http://observer.theguardian.com">The Observer</a>
+                @content.content.blogOrSeriesTag.map { series =>
+                    <a class="submeta2__link" href="@LinkTo {/@series.id}">@series.name</a>
+                }.getOrElse {
+                    @if(content.content.isFromTheObserver) {
+                        <a class="submeta2__link" href="http://observer.theguardian.com">The Observer</a>
+                    }
                 }
-            }
+            </div>
         </div>
 
         <div class="submeta2__keywords">
-            @if(content.tags.keywords.filterNot(_.isSectionTag).nonEmpty) {
-                @defining(content.tags.keywords.filterNot(_.isSectionTag).slice(1, 6)) { shownKeywords =>
-                    @if(shownKeywords.nonEmpty) {
-                        @shownKeywords.zipWithRowInfo.map{ case(keyword, row) =>
-                            <a class="submeta2__link @if(row.rowNum == 1){submeta2__link--first}"
-                                href="@LinkTo(keyword.metadata.url)"
-                                data-link-name="keyword: @keyword.id">
-                                    @keyword.name
-                                    @if(content.tags.keywords.filter(_ != keyword).find(_.name == keyword.name)){ (@keyword.properties.sectionName) }
-                            </a>
+            <div class="submeta2__links">
+                @if(content.tags.keywords.filterNot(_.isSectionTag).nonEmpty) {
+                    @defining(content.tags.keywords.filterNot(_.isSectionTag).slice(1, 6)) { shownKeywords =>
+                        @if(shownKeywords.nonEmpty) {
+                            @shownKeywords.zipWithRowInfo.map{ case(keyword, row) =>
+                                <a class="submeta2__link @if(row.rowNum == 1){submeta2__link--first}"
```

Comment:
Since you removed `submeta2__link--first`, best to remove all the logic that adds the class too 😄